pkg/kubelet/cloudresource/cloud_request_manager.go

```diff
@@ -32,8 +32,8 @@ import (
 
 // SyncManager is an interface for making requests to a cloud provider
 type SyncManager interface {
-	Run(stopCh <-chan struct{})
-	NodeAddresses() ([]v1.NodeAddress, error)
+	Run(ctx context.Context, stopCh <-chan struct{})
+	NodeAddresses(context.Context) ([]v1.NodeAddress, error)
```

Comment:
nit: for the interface to look consistent
```suggestion
	NodeAddresses(ctx context.Context) ([]v1.NodeAddress, error)
```

Reply:
Will fix